### ISSUE TITLE
[Bug]: Tabs are missing the filename

### ISSUE THREAD

**Issue by DanielHilton** [mallowigi/material-theme-issues#345]
### Describe the bug

When enabling material theme UI in Goland 2023.1.1 the tab names no longer appear.

### To Reproduce

1. Open Goland
2. Material Theme UI v8.6.3

### Expected behavior

File names appear in the tab

### Material Theme Version

v8.6.3

### IDE

Other (AppCode, CLion, GoLang...)

### System Info

```shell
GoLand 2023.1.1
Build #GO-231.8770.71, built on May 2, 2023
Licensed to ****
Subscription is active until February 1, 2024.
Runtime version: 17.0.6+10-b829.9 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.2
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 10
Metal Rendering is ON
Registry:
    editor.focus.mode.color.light=5b595c
    ide.editor.tab.selection.animation=true
    suggest.all.run.configurations.from.context=true
    ide.completion.variant.limit=500
    ide.new.editor.tabs.vertical.borders=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=5b595c

Non-Bundled Plugins:
    com.github.patou.gitmoji (1.12.1)
    com.github.copilot (1.2.5.2507)
    hu.hawser.co-author-plugin (2.3)
    com.intellij.kubernetes (231.8770.3)
    monokai-pro (1.10)
    com.mallowigi (90.0.0)
    mobi.hsz.idea.gitignore (4.5.0)
    com.chrisrm.idea.MaterialThemeUI (8.6.3)
    net.ashald.envfile (3.4.1)
```

### Logs/Screenshots

<img width="590" alt="image" src="https://user-images.githubusercontent.com/28859662/236152961-a71f1199-c4d6-4339-9166-ad63d1cce3b0.png">

**Comment by mallowigi** [comment]
Hopefully this is fixed in 8.6.4

**Comment by DanielHilton** [comment]
> Hopefully this is fixed in 8.6.4

Updated and can confirm it is fixed. thanks 🎉
